src/App.vue

```diff
@@ -307,21 +330,17 @@ export default class App extends Mixins(mixins.TransactionMixin, NodeErrorMixin)
       // connection to node
       await this.runAppConnectionToNode();
     });
-    updateDocumentTitle(); // For the first load
```

Comment:
Why was it removed?

Reply:
Called previously in `setLanguage` action